src/mpid/ch4/netmod/ofi/ofi_capability_sets.h

```diff
@@ -45,7 +46,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_get_set_number(char *set_name)
     } else if (!strcmp("verbs", set_name)) {
         return MPIDI_OFI_SET_NUMBER_VERBS;
     } else {
-        return -1;
+        return MPIDI_OFI_SET_NUMBER_UNKNOWN;
```

Comment:
here "unknown" is more correct (and intuitive :) ), but for caps indexing "minimal" is easy to understand. I can add "minimal" constant which has same value as "unknown".
What do you think?